packages/editor-ui/src/stores/collaboration.store.ts

```diff
@@ -0,0 +1,49 @@
+import { defineStore } from 'pinia';
+import { computed, ref } from 'vue';
+import type { ActiveUsersForWorkflows } from '@/Interface';
+import { useWorkflowsStore } from '@/stores/workflows.store';
+import { usePushConnectionStore } from '@/stores/pushConnection.store';
+import { STORES } from '@/constants';
+
+export const useCollaborationStore = defineStore(STORES.COLLABORATION, () => {
+	const pushStore = usePushConnectionStore();
+	const workflowStore = useWorkflowsStore();
+
+	const usersForWorkflows = ref<ActiveUsersForWorkflows>({});
```

Comment:
IMO the way we have a single landfill file for all types is not very maintainable. Hence maybe we could move the `ActiveUsersForWorkflows` type into this file, as this is where it's used